### ISSUE TITLE
Unable to login to Web portal

### ISSUE THREAD

**Issue by jamilahsan** [Raikia/FiercePhish#93]
I have installed and configured FiercePhish. When I try to connect to http://my-public-ip:80 I am prompted for user name and password.
When I enter admin as user name and my password which I entered in the configuration file, I get "Whoops, looks like something went wrong". Any suggestion - Idea?

**Comment by mother2110** [comment]
I get the same thing

**Comment by Raikia** [comment]
Look in /var/www/fiercephish/storage/logs/laravel.log, that will direct you to what is wrong.  It sounds like a service isn't starting (like mysql) or something causing that issue.  Double check your mysql creds in /var/www/fiercephish/.env and make sure your system has enough RAM to run mysql

**Comment by mother2110** [comment]
[2022-08-17 18:32:59] master.ERROR: ErrorException: dns_get_record(): A temporary server error occurred. in /var/www/fiercephish/app/Libraries/DomainTools.php:75

**Comment by mother2110** [comment]
SQLSTATE[HY000] [1045] Access denied for user 'fiercephish'@'localhost' (using password: YES) in /var/www/fiercephish/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/PDO/Exception.php:18

**Comment by Raikia** [comment]
Yeah sounds like your mysql password is incorrect, check /var/www/fiercephish/.env and maybe try manually logging into the mysql server using the "mysql" cli

**Comment by mother2110** [comment]
root@kphisher:~# mysqladmin -u root -p status
Enter password:
Uptime: 381  Threads: 2  Questions: 17  Slow queries: 0  Opens: 169  Flush tables: 3  Open tables: 88  Queries per second avg: 0.044


password does work

**Comment by Raikia** [comment]
that's logging in as root.  If you want to do that, update the /var/www/fiercephish/.env DB_USERNAME and DB_PASSWORD variables to make sure its using the root user, then run "cd /var/www/fiercephish/; php artisan config:cache"

**Comment by mother2110** [comment]
I think I got it. I was reusing the same config, and I think that was causing the issue. I rebuilt the server and re-ran the install and it works fine
